presto-main/src/test/java/io/prestosql/sql/planner/TestPredicatePushdown.java

```diff
@@ -52,9 +56,12 @@
 {
     private final Metadata metadata = createTestMetadataManager();
 
-    TestPredicatePushdown()
+    public TestPredicatePushdown()
     {
-        super(ImmutableMap.of(ENABLE_DYNAMIC_FILTERING, "true"));
+        super(ImmutableMap.of(
+                JOIN_REORDERING_STRATEGY, JoinReorderingStrategy.NONE.name(),
+                JOIN_DISTRIBUTION_TYPE, JoinDistributionType.PARTITIONED.name(),
```

Comment:
Remove these and try to fix the two failing tests by updating the expected plans.

Reply:
Done.